epsagon/modules/redis.py

```diff
@@ -44,6 +44,12 @@ def patch():
         'Redis.execute_command',
         _single_wrapper
     )
+    # Version < 3.0
+    wrapt.wrap_function_wrapper(
```

Comment:
to make sure, in newer versions this call won't explode - right?

Reply:
That's right, this code is failsafe